quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/StartCommandDistTest.java

```diff
@@ -139,15 +139,25 @@ void failIfOptimizedUsedForFirstStartup(CLIResult cliResult) {
     }
 
     @Test
-    @Launch({ "start", "--http-enabled=true" })
+    @Launch({ "start", "--db=dev-file", "--http-enabled=true" })
     void failNoHostnameNotSet(CLIResult cliResult) {
         assertTrue(cliResult.getErrorOutput().contains("ERROR: hostname is not configured; either configure hostname, or set hostname-strict to false"),
                 () -> "The Output:\n" + cliResult.getOutput() + "doesn't contains the expected string.");
     }
 
+    @RawDistOnly(reason = "Containers are immutable")
+    void testFastStartOptimized(KeycloakDistribution dist) {
```

Comment:
Missing `@Test` annotation.

We don't have tests for fast optimized start? Or is it really necessary to have it here?

Reply:
It's no longer necessary here, there's coverage in PicocliTest - there were other refactorings so we now exercise fast start in a unit testable manner.